tests/accessor/accessor_common.h

```diff
@@ -358,14 +438,16 @@ void read_write_acc(AccT testing_acc, ResultAccT res_acc) {
  * @param r Range for accessors buffer
  */
 template <accessor_type AccType, typename DataT, int Dimension,
-          sycl::access_mode AccessMode, sycl::target Target,
+          sycl::access_mode AccessMode,
+          sycl::target Target = sycl::target::device,
```

Comment:
Looks like we need the similar change for `check_zero_dim_constructor`

Reply:
Fixed